server/middlewares/BorrowValidator.js

```diff
@@ -0,0 +1,31 @@
+import BookValidator from './BookValidator';
+import checkForErrors from './checkForErrors';
+
+/**
+ * @class BorrowValidator
+ * @classdesc Validates the borrowing a book route
+ */
+export default class BorrowValidator {
+  /**
+   * Generic Number validator
+   * @param {string} item
+   * @returns {function} call to a check API middleware
+   * @memberof Validation
+   */
+  static checkNumber(item) {
+    return BookValidator.checkNumber(item);
+  }
```

Comment:
you can just call the book validator down. you can remove this.

Reply:
Thank you, noted